improver_tests/acceptance/test_spot_extract.py

```diff
@@ -501,3 +501,22 @@ def test_percentile_from_threshold_with_realizations(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path)
+
+
+def test_local_timezone_extraction(tmp_path):
+    """Test spot extraction of diagnostic on a global grid representing local
+    timezone information. The input cube contains a 2-dimensional time
+    coordinate, from which a time for each site should be extracted."""
+    kgo_dir = acc.kgo_root() / "spot-extract"
+    neighbour_path = kgo_dir / "inputs/all_methods_global.nc"
+    diag_path = kgo_dir / "inputs/engluk_temperature_max_localtime.nc"
+    kgo_path = kgo_dir / "outputs/engluk_localtime_kgo.nc"
```

Comment:
I have set the title using the --new-title option so that it matches the expected (as is done in the suite).
The forecast_period is missing and should be missing. I am going to have to create a PR to update the metadata interpreter with handling for this and cell_methods that contain comments excluding the `_in_vicinity` suffix. It does not need to be done now (though I've started it). I'll write a ticket this afternoon.